proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -66,7 +62,9 @@ message MsgCreateValidator {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable)   = false
   ];
-  string                   delegator_address = 4 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Deprecated: Use of Delegator Address in MsgCreateValidator is deprecated.
+  // The validator address and delegator address are always same while creating validator.
```

Comment:
The strings, i.e. Bech32, are not the same. The underlying bytes are. I think we should make this clear.